CMakeLists.txt

```diff
@@ -36,11 +38,14 @@ target_compile_definitions(
 set_target_properties(
     tinyxml2
     PROPERTIES
-    DEFINE_SYMBOL "TINYXML2_EXPORT"

```

Comment:
This change is irrelevant. The `DEFINE_SYMBOL` property "sets the name of the preprocessor symbol defined when compiling sources in a shared library". Thus there is no reason to make this change (including the `if(BUILD_SHARED_LIBS)` addition below)

Read the docs:

Reply:
[`093dd7a` (#922)](https://github.com/leethomason/tinyxml2/pull/922/commits/093dd7a1fe484c58c63a0e7b5a9a5f08b4efef5e)